src/network/network_coordinator.cpp

```diff
@@ -229,6 +343,118 @@ void ClientNetworkCoordinatorSocketHandler::GetListing()
 	this->SendPacket(p);
 }
 
+/**
+ * Join a server based on an invite code.
+ * @param invite_code The invite code of the server to connect to.
+ * @param connecter The connecter of the request.
+ */
+void ClientNetworkCoordinatorSocketHandler::ConnectToServer(const std::string &invite_code, TCPServerConnecter *connecter)
+{
+	assert(invite_code[0] == '+');
+
+	if (this->connecter_pre.find(invite_code) != this->connecter_pre.end()) {
+		/* If someone is hammering the refresh key, he can set out two
```

Comment:
We eradicated the (s)he from the code comments, also is `set out` what you meant? 
So something like `..., one can sent out ...`?